### PR TITLE
[dataquery] Removed last instance of getJS.php

### PR DESCRIPTION
**Pull request by ridz1208**
### Brief summary of changes
GetJS.php no longer exists and throws a 404 error on current production environments

### To test this change...

- [ ] when attempting to download data fron the DQT as zip, you get a 404 without this PR

### CODE REVIEW

**Comment by ridz1208** [comment]
@johnsaigle could you review ?

File: modules/dataquery/jsx/react.tabs.js [hunk 1]
```diff
@@ -213,7 +213,7 @@ var ViewDataTabPane = React.createClass({
         window.dataBlobs = [];
 
         if(FileList.length < 100 || confirm("You are trying to download more than 100 files. This may be slow or crash your web browser.\n\nYou may want to consider splitting your query into more, smaller queries by defining more restrictive filters.\n\nPress OK to continue with attempting to download current files or cancel to abort." )) {
-            saveworker = new Worker(loris.BaseURL + '/GetJS.php?Module=dataquery&file=workers/savezip.js');
+            saveworker = new Worker(loris.BaseURL + '/dataquery/js/workers/savezip.js');
```
Comment (driusan) [inline modules/dataquery/jsx/react.tabs.js]: why is the compiled file above (can't comment on it) still part of this branch? Shouldn't it be generated by make?

Comment (ridz1208) [reply modules/dataquery/jsx/react.tabs.js]: This is going to bugfix on version 20.3.0, I'd rather have it in v20.3.1 given the difficulty for projects to go to 21 and the need for this to use the DQT

**Comment by driusan** [comment]
This is a simple and obvious enough bug fix that I think we just need someone to test it, not review it..

**Comment by johnsaigle** [comment]
@ridz1208 Code changes look fine but I don't really have any experience with DQT. Actually I don't even have it set up on my sandbox.

May be better to have someone else test this for you. Sorry about that.

**Pull request merged**